src/routes/view-content/+page.svelte

```diff
@@ -268,6 +268,8 @@
             openGuideMenu();
         } else if (tab === 'guide' && $selectedBibleSection === null) {
             openGuideMenu();
+        } else if (tab === 'guide' && stepsAvailable.length === 0) {
```

Comment:
This mimics how the page currently works: if there are no cbbter steps, we show the guide menu.